keylime/tenant.py

```diff
@@ -569,12 +569,14 @@ def do_cv(self):
             logger.error("Verifier at %s with Port %s timed out.", self.verifier_ip, self.verifier_port)
             sys.exit()
 
+        response_json = Tenant._jsonify_response(response, False, True)
+
         if response.status_code == 409:
             # this is a conflict, need to update or delete it
-            Tenant._print_json_response(response)
+            json.dumps(response_json)
```

Comment:
`print(json_response);` ?

Reply:
Fair enough, keeping exactly the same functionality as before, I will change.